tmail-backend/apps/distributed-es6-backport/pom.xml

```diff
@@ -524,12 +524,8 @@
                             <to>
                                 <image>linagora/tmail-backend</image>
                                 <tags>
-                                    <tag>distributed-esv6-${env.DOCKER_TAG}</tag>
+                                    <tag>distributed-esv6-branch-master</tag>
```

Comment:
Should keep this at the end. As we have tag when release, not only master tag.

Reply:
I Override custom tag in Jenkinsfile, see it in the last commit